lib/src/slider/Slider.jsx

```diff
@@ -51,7 +52,11 @@ const DxcSlider = ({
   return (
     <ThemeProvider theme={colorsTheme.slider}>
       <SliderContainer margin={margin} size={size}>
-        {showLimitsValues && <MinLabelContainer disabled={disabled}>{minValue}</MinLabelContainer>}
+        {showLimitsValues && (
+          <MinLabelContainer disabled={disabled}>
+            {labelFormatCallback ? labelFormatCallback(minValue) : minValue}
```

Comment:
@marcialfps, can we use memoized values when the callback and the minValue/maxValue remain unchanged? (with useMemo)

Reply:
Added memoization in the last commit. Thanks @jsardid !